packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/features/exposurepoint/ExposurePointFeature.java

```diff
@@ -63,16 +67,21 @@ public void updateBuilder(CaptureRequest.Builder requestBuilder) {
     if (!checkIsSupported()) {
       return;
     }
-
-    MeteringRectangle aeRect = null;
-    try {
-      aeRect = cameraRegions.getAEMeteringRectangle();
-    } catch (Exception e) {
-      Log.w("Camera", "Unable to retrieve the Auto Exposure metering rectangle.", e);
-    }
-
     requestBuilder.set(
         CaptureRequest.CONTROL_AE_REGIONS,
-        aeRect == null ? null : new MeteringRectangle[] {aeRect});
+        exposureRectangle == null ? null : new MeteringRectangle[] {exposureRectangle});
+  }
+
+  private void buildExposureRectangle() {
+    if (!checkIsSupported()) {
+      return;
+    }
```

Comment:
See also previous comment for the `checkIsSupported()` method. I feel it would be better to check if the `cameraBoundaries` has been set and if not throw an `AssertionError`. This will inform developers that they should set the `cameraBoundaries` before updating the feature value.

Maybe something like:
```suggestion
    if (cameraBoundaries == null) {
      throw new AssertionError("The cameraBoundaries should be set (using `ExposurePointFeature.setCameraBoundaries(Size)`) before updating the exposure point.");
    }
```